packages/web-components/package.json

```diff
@@ -49,10 +52,9 @@
   "devDependencies": {
     "@microsoft/eslint-config-fast-dna": "^1.2.0",
     "@microsoft/api-extractor": "7.7.1",
-    "@storybook/addons": "^5.3.8",
-    "@storybook/cli": "^5.3.8",
-    "@storybook/html": "^5.3.8",
-    "@storybook/theming": "^5.3.8",
+    "@storybook/addons": "^6.1.17",
```

Comment:
hey folks! would you be satisfied to use storybook `6.0.28` for now ? We plan to migrate to latest in June https://github.com/microsoft/fluentui/issues/18142 so we could be all on the same version -> getting close to single version policy for tooling in monorepo + shorter installation times 

pls lemme know, thx

Reply:
I'm fine with this, though I think this change is likely coming from rebasing the feature branch w/ master. We'll likely need to make this change in a new PR.

@bheston can you rebase w/ the feature branch? That should clean up the commit history and files here :)